renku/data/shacl_shape.json

```diff
@@ -408,7 +408,7 @@
                   "@id": "xsd:string"
                },
                "maxCount": 1,
-               "sh:pattern": "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\+\\d{2}:\\d{2}"
+               "sh:pattern": "\\d{4}-\\d{2}-\\d{2}(T\\d{2}:\\d{2}:\\d{2}\\+\\d{2}:\\d{2})?"
```

Comment:
getting yyyy-MM-dd with the current SHACL (without this fix) would cause triples generation to fail. Does this actually happen in the wild? If so, we'd probably need a bugfix release for this, no?